ops/scripts/generate_config.sh

```diff
@@ -59,6 +59,20 @@ fi
 echo "Sourcing secrets"
 source "./.secrets"
 
+TARGET_FILE=".env.test"
+VARS=("GIGADB_USER" "GIGADB_HOST" "GIGADB_DB" "GIGADB_PORT")
+
+> "$TARGET_FILE"
+
+for VAR in "${VARS[@]}"; do
+    if [ "${!VAR+x}" ]; then
+        echo "$VAR=${!VAR}" >> "$TARGET_FILE"
+    else
+        echo "Variable $VAR not found"
+    fi
+done
+
+echo "Variables added to $TARGET_FILE"
```

Comment:
**question:** In the `api.suite.yml`, you have made use of the variable `GIGADB_PASSWORD`, but why the value is not retrieved into the .env.test file?
**suggestion:** Better include `GIGADB_PASSWORD` in the above VARS array, although the test will pass with or without it, this will make the suite file consistent.